src/App.js

```diff
@@ -16,6 +17,8 @@ function App() {
           <Route exact path='/' element={<LandingPage />} />
           <Route path='/Admin' element={<AdminPage />} />
           <Route path='/Submission' element={<SubmissionForm />} />
+          <Route path='/SubmissionDetails' element={<SubmissionDetailsPage />}
```

Comment:
I think we'll want to make this a dynamic route that looks like path='/Admin/:id' and create Links to that route for each submission card on the admin page. 

I think it's fine to leave here for now though so that we can see it, we can move it once we build out more functionality.